styles.css

```diff
@@ -3,6 +3,7 @@
   font-family: 'Roboto', sans-serif;
   color: #fff;
   text-decoration: none;
+  margin: 0;
 }
 
 body {
```

Comment:
- [ ] The section should be `display: flex` as the requirement says see the attached screen shot in the other comment.